lambda/chain.json

```diff
@@ -96,12 +104,20 @@
       {
         "address": "grpc.lambda.nodestake.top:443",
         "provider": "NodeStake"
+      },
+      {
+        "address": "lambda-grpc.noders.services:31090",
+        "provider": "[NODERS]TEAM"
       }
     ],
     "evm-http-jsonrpc": [
       {
         "address": "https://jsonrpc-lambda.d-stake.xyz",
         "provider": "D-stake"
+      },
+      {
+        "address": "https://lambda-jsonrpc.noders.services",
+        "provider": [NODERS]TEAM"
```

Comment:
Missing starting comma for provider name here

Reply:
Corrected